### PR TITLE
New env

### PR DESCRIPTION
**Pull request by toshiakitakase**

### CODE REVIEW

File: frontend/packages/react-app/.env.example [hunk 1]
```diff
@@ -1,5 +1,6 @@
-REACT_APP_RPC_URL_1=
-REACT_APP_RPC_URL_4=
+REACT_APP_RPC_URL=
 REACT_APP_FORTMATIC_API_KEY=
 REACT_APP_PORTIS_DAPP_ID=
 REACT_APP_TARGETS_UPPER_LIMIT=2000
+REACT_APP_SUBGRAPH=
+REACT_APP_CHAIN_ID=
```
Comment (wildmouse) [inline frontend/packages/react-app/.env.example:6]: could you add following envs to env.example? 

```
REACT_APP_CONTRACT_MIGRATIONS,
REACT_APP_CONTRACT_WALLETDISTRIBUTOR,
REACT_APP_CONTRACT_UUIDDISTRIBUTOR,
REACT_APP_CONTRACT_CCTWALLETDISTRIBUTOR,
```

Comment (wildmouse) [reply frontend/packages/react-app/.env.example:6]: (without values)

Comment (toshiakitakase) [reply frontend/packages/react-app/.env.example]: Sorry for my mistake!

**Comment by wildmouse** [comment]
An available network is not specified, so users cant understand which is a correct network. 

Screen Shot 2021-02-26 at 14 59 20

**Comment by wildmouse** [comment]
$LINK address of rinkeby is hard coded, and this address is not necessary anymore. 
It had to be deleted at Audius code deletion PR, so I will delete this in this PR. Sorry. 

https://github.com/iroiro/iroiro/blob/f2563caf2820ea90e3171d9e2b83c84746d1ec79/frontend/packages/react-app/src/components/pages/TokenCampaignDetailPage.tsx#L66-L70

**Comment by wildmouse** [comment]
Fixed!

**Comment by toshiakitakase** [comment]
Added support chain id!

**Review by wildmouse — APPROVED** [review]
LGTM 🏃‍♂️ 
Its working on both Rinkeby and Kovan testnet😄

**Pull request merged**